swagger.md

```diff
@@ -0,0 +1,15 @@
+#Oaze.io - Frontend Dev Task
+
+We would like to learn about how you approach a developement. In order to do that we have prepares a small task based on our initial MVP mocks. 
```

Comment:
Minor typos:
developement -> development
prepares -> prepared

Reply:
👍 ty :-) more typos on the way 🗡